CRM/Core/BAO/RecurringEntity.php

```diff
@@ -173,17 +153,15 @@ public static function add(&$params) {
    * @param string $entityTable
    *   Name of the entity table .
    *
-   *
-   * @return object
+   * @return CRM_Core_DAO_RecurringEntity
    */
   public static function quickAdd($parentId, $entityId, $entityTable) {
```

Comment:
Isn't this just so much handier....

Reply:
It's pretty dumb. I considered deleting it.